src/ashell/comms-shell.c

```diff
@@ -80,10 +80,6 @@ static void insert_char(char *pos, char c, uint8_t end)
 {
     char tmp;
 
-    /* Echo back to console */
```

Comment:
Rather than printing here one character at a time, I now use comms_print_strncpy to print all text in the uart that hasn't been echoed yet.  This prevents duplicated characters and improves performance when pasting large amounts of text.